host/js/VERSION

```diff
@@ -1 +1 @@
-3.0.0-alpha.14
\ No newline at end of file
+3.0.0-beta.1
```

Comment:
question: why this bump?

Reply:
it shouldn't. I accidentally commited it.